### ISSUE TITLE
Support for react-hyperscript/React.createElement in React Specimen

### ISSUE THREAD

**Issue by nickjohnson-dev** [interactivethings/catalog#328]
Hi,

First of all, this is a great tool, so thanks for the work you've put into it!

I am in charge of a set of React components distributed through my company, and maintain a documentation site for these components. Currently the documentation is all a custom React app, but I think Catalog would be a great fit for this purpose.

At my company, we utilize [react-hyperscript](https://github.com/mlmorg/react-hyperscript) instead of JSX for our React components, and I need to provide examples within the documentation using that syntax.

Currently, the cleanest way I can get this to work is to do this:

```
` ` `react
const Demo = h('button', {}, ['Click Me']);
<Demo/>
` ` `
```

However I'd love to be able to do this more directly:

```
` ` `react
h('button', {}, ['Click Me']);
` ` `
```

I understand that this might be difficult with the current architecture, judging from issue [#149](https://github.com/interactivethings/catalog/issues/149), but I just wanted to see if this was something that could be supported, possibly just through an API that allows disabling of the JSX transform being run on the contents of the Specimen.

I am also open to creating a custom Specimen, a possibility alluded to in issue [#128](https://github.com/interactivethings/catalog/issues/128), even if the API is unstable. I would however need some brief direction on how to use the API.

Anyway, thanks again. Please let me know if anything above is unclear.

**Comment by herrstucki** [comment]
Hi! Thanks for the kind words and for reporting this issue. This should be solved in #329. Can you confirm that this works for you (check out that branch, run `make` and `yarn link` to test)?

By the way, you can also write your documentation using JS/React as described in https://docs.catalog.style/guides/react#markdown. This would already allow you to do this for example:

```jsx
// MyComponent.docs.js

import h from 'react-hyperscript';
import React from 'react';
import {markdown, ReactSpecimen} from 'catalog';
export default () => markdown`
# Hello

${<ReactSpecimen>
  {h('button, {}, ['Click Me'])}
</ReactSpecimen>}
`
```

**Comment by nickjohnson-dev** [comment]
Thanks for the quick response! I will try this out as soon as I am back in front of my computer. :)

**Comment by nickjohnson-dev** [comment]
Sorry for the delay in being able to test this, but it works like a charm!

Nice work, and thanks again! :)

**Comment by herrstucki** [comment]
@nickjohnson-dev Released in 3.1.5 😄
